release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2857,6 +2857,49 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.7
+[id="ocp-4-17-7_{context}"]
+=== RHSA-2024:10518 - {product-title} {product-version}.7 bug fix, and security update advisory
+
+Issued: 03 December 2024
+
+{product-title} release {product-version}.7 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:10518[RHSA-2024:10518] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:10521[RHBA-2024:10521] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.7 --pullspecs
+----
+
+[id="ocp-4-17-7-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-7-cluster-tasks-pipelines_{context}"]
+===== Deprecated clusterTasks OpenShift Pipelines version 1.17
+
+* The {product-title} {product-version} release deprecates the `clusterTasks` resource from {pipelines-title} version 1.17. The release also removes `clusterTasks` resource dependencies from the static plugin on the {pipelines-shortname} page of the {product-title} web console. (link:https://issues.redhat.com/browse/OCPBUGS-44183[*OCPBUGS-44183*])
+
+[id="ocp-4-17-7-known-issues_{context}"]
+==== Known issues
+
+// SME review
+* When you attempt to use the the Cluster Network Operator (CNO) to upgrade your cluster from {product-title} 4.16 to {product-title} {product-version}, `ovnkube-control-plane` pods fail to run because `localnet` topology networks do not have subnets defined. This issue causes the upgrade operation to pause and outputs an error message for the `secondaryNetworkClusterManager` resource, which was created by the cluster controller manager. As a workaround for this issue, add a subnet to the Network Attachment Definitions (NAD) section of the `secondaryNetworkClusterManager` resource YAML file so that the cluster upgrade operation can continue as expected. (link:https://issues.redhat.com/browse/OCPBUGS-43454[*OCPBUGS-43454*])       
```

Comment:
```suggestion
* When you attempt to use the the Cluster Network Operator (CNO) to upgrade your cluster from {product-title} 4.16 to {product-title} {product-version}, `ovnkube-control-plane` pods fail to run when the cluster features secondary ovn-kubernetes `localnet` topology networks without subnets defined. This issue causes the upgrade operation to pause and outputs an error message for the `secondaryNetworkClusterManager` resource, which was created by the cluster controller manager. (link:https://issues.redhat.com/browse/OCPBUGS-43454[*OCPBUGS-43454*])
```

The workaround listed there should not be considered valid - it might cause more harm than good if the network is in use - it'll cause the workloads to have to be removed, then re-added.